### ISSUE TITLE
Onboard App Analyst

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#438]
Our new app analyst starts on 8/19!

**Comment by johnclary** [comment]
@dianamartin i talked to surbhi about hardware and she's going to with a macbook. we have a new one that mateo can dig up for you.

**Comment by dianamartin** [comment]
@johnclary Yeah, Mateo asked me where it was and we I found the macbook in our DTS storage room.

**Comment by johnclary** [comment]
@dianamartin do we need to get a chair?

**Comment by dianamartin** [comment]
@johnclary we have a guest chair, which is the same type that Tracy uses. I also grabbed the Allandale conference chair that was in Koenig (temporarily).

**Comment by dianamartin** [comment]
@johnclary Scott has possession of the chair, it will be en route to Cameron today. He has an SUV today and doing us a solid.

**Comment by johnclary** [comment]
@dianamartin would you make sure the remaining Accounts get set up for surbhi?
